public/scripts/main.js

```diff
@@ -256,23 +256,99 @@ hitsTheBooks.controller('accountAccessController', function($scope, $rootScope,
 
   // Login
   $scope.loginData = { username: '', password: '' };
-
+  $scope.signinAlert = 0;
```

Comment:
I would suggest using an enum for the value of signinAlert, just to make it clearer what each case means